cmd/virt-handler/virt-handler.go

```diff
@@ -150,6 +151,22 @@ func (app *virtHandlerApp) Run() {
 		panic(err)
 	}
 
+	// Copy the tap device maker binary
+	if err = copy("/usr/bin/tap-device-maker", filepath.Join(app.VirtLibDir, "tap-device-maker")); err != nil {
+		panic(err)
+	}
+	// relabel the binary to container_file_t (no categories) so the handler can execute it
+	relabelTapMakerBinaryCmdArgs := []string{
+		"--mount", "/proc/1/ns/mnt",
+		"exec", "--",
+		"/usr/bin/chcon", "-t", "container_file_t", "-l", "s0",
```

Comment:
Let's do this via native selinux code in `virt-chroot` instaed of using the `chcon` binary.

Reply:
Done in https://github.com/kubevirt/kubevirt/pull/3290/commits/c9a4c9ff8fba68a9b74a5fa28d13373aeffb7d9c